lib/Message.php

```diff
@@ -81,7 +81,7 @@ function getBodyAsString() : string {
             return ob_get_clean();
         }
         $contentLength = $this->getHeader('Content-Length');
-        if (null === $contentLength) {
+        if (null === $contentLength || empty($contentLength)) {
```

Comment:
I guess this is less error-prone to set `$contentLength` to `0` if empty. The bug could appear somewhere else later.